Description: Replace not packaged yaml-rust2 with its origin yaml-rust
Forwarded: not-needed
Author: Blair Noctis <ncts@debian.org>
Last-Update: 2025-03-04
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -207,4 +207,5 @@
 [dependencies.yaml-rust2]
-version = "0.10"
+version = "0.4"
 optional = true
+package = "yaml-rust"
 
